modules/gas-estimate/lib/owlracle.service.ts

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios';
+import _ from 'lodash';
+import { env } from '../../../app/env';
+import { GqlGasEstimatesData } from '../../../schema';
+
+interface OwlracleRequest {
+    apikey: string;
+    blocks: number;
+    percentile: number;
+    accept: string;
+    version: number;
+}
+export class OwlracleService {
+    private readonly baseUrl: string;
+    private readonly requestParams: OwlracleRequest;
+
+    constructor() {
+        this.baseUrl = 'https://owlracle.info/ftm/gas';
```

Comment:
make this url configurable, ie: what if this needed to work for a different chain?

Reply:
@danielmkm I have updated the PR to make it multi chain. If it's ok I'll need to update the FE implementation too.